content/challenges/medium/11 - Identical Sub Arrays/func.js

```diff
@@ -13,16 +13,15 @@ module.exports = (arr) => {
     let numSameArr = 0;
 
     for (let i = 0; i < arr.length; i++) {
-        if (arr[i].length === 0) {
+        if (arr[i].length === 0 || arr[i].length === 1) {
```

Comment:
Arrays can't have negative lengths so how else could you write this instead of using two expressions?

think about using one of the following:
* `>`
* `<`
* `>=`
* `<=`